### PR TITLE
Ebnt 189 motivation

### PR DESCRIPTION
**Pull request by VlasovKirill**
Add motivation to docs and readme

### CODE REVIEW

**Comment by i-a-andrianov** [comment]
Please fix `MANIFEST.in` to ensure that CI checks pass

File: README.rst [hunk 1]
```diff
@@ -67,9 +67,31 @@
...
+- Integrate your models into existing infrastructure
+- Wrap the model and run as a service
+- Track Model Metrics and Need to follow degradation of models
+- Provide fast delivery of models in production
+- An easy way to test hypotheses
+
+You need Ebonite if
+===================
+- Have more than 1 ML models
+- Need to new models appear
```
Comment (mike0sv) [inline README.rst]: ???

File: README.rst [hunk 2]
```diff
@@ -67,9 +67,31 @@
...
+- Need to Accelerates prototyping
```
Comment (mike0sv) [inline README.rst]: ????????

File: README.rst [hunk 3]
```diff
@@ -67,9 +67,31 @@
...
+- Need to re-train and update often
```
Comment (mike0sv) [inline README.rst]: ????

File: README.rst (hunk 1 above)
Comment (mike0sv) [reply README.rst]: need for?

File: README.rst [hunk 4]
```diff
@@ -67,9 +67,31 @@
...
+- Need to train multistaging models
```
Comment (mike0sv) [inline README.rst]: multistage?

File: README.rst [hunk 5]
```diff
@@ -67,9 +67,31 @@
...
+- Have several team working on different models
```
Comment (mike0sv) [inline README.rst]: teams?

**Pull request closed without merging**